Python/fibonacci.py

```diff
@@ -0,0 +1,13 @@
+# Fibonacci Sequence:
+# Write a Python function to generate the Fibonacci sequence up to a given number of terms.
+def fibonacci(n):
+    fib_sequence = [0, 1]
+    while len(fib_sequence) < n:
```

Comment:
If I want Fib sires of 1 term then also this would be producing 2 outputs atleast.
Not an issue but you can think how to do that.